tools/base/fpgaconf/bitstream-tools.h

```diff
@@ -168,58 +169,65 @@ fpga_result check_bitstream_guid(const uint8_t *bitstream)
 	return FPGA_OK;
 }
 
-fpga_result get_fpga_interface_id(fpga_token token, uint64_t *id_l,
-								uint64_t *id_h)
+int64_t bs_int64_be_to_le(int64_t val)
```

Comment:
Consider using compiler builtin function for byte swapping?

Reply:
http://man7.org/linux/man-pages/man3/bswap.3.html
```
       #include <byteswap.h>

       bswap_16(x);
       bswap_32(x);
       bswap_64(x);
```

Reply:
fixed with bswap_64(x)